### ISSUE TITLE
v0.3.0: level-item now justifies content to center, leading to ugly mobile view

### ISSUE THREAD

**Issue by davidmathlogic** [jgthms/bulma#466]
### Checklist

* This is about the Bulma **CSS framework**
* I'm using Bulma **version** 0.3.0
* My **browser** is: Chrome

### Description

A lot of new styling seems to have been added to level-item in 0.3.0. 

### My HTML

```<div class="level">
    <div class="level-left">
      <div class="level-item">
        <h1 class="title is-4">Teaching</h1>
      </div>
    </div>
    <div class="level-right">
      <div class="level-item">
        <a class="button">Past Teaching</a>
      </div>
      <div class="level-item">
        <a class="button">Resources</a>
      </div>
    </div>
  </div>
```

### Expected behavior

I expected all content in level-items to align to the left margin by default the way it did in 0.2.3: 

![oldbehavior](https://cloud.githubusercontent.com/assets/18088182/21959545/4a60e08c-da98-11e6-86da-473441dd14a2.png)

### Actual behavior

The content is center justified, leading to an unaesthetic asymmetrical mobile view: 

![newbehavior](https://cloud.githubusercontent.com/assets/18088182/21959546/4f548788-da98-11e6-8727-9ba23981e0de.png)

**Comment by grebebird** [comment]
Yeah, I've given up on using level-item because of this. The centering hasn't been becoming in the situations we've run into. Was looking for if `.level-item` is even really _needed_ inside of `.level` when I found this post and found I wasn't alone in my feeling about this choice.
